src/nft/pages/asset/Asset.tsx

```diff
@@ -38,9 +35,9 @@ const AssetPriceDetailsContainer = styled.div`
   }
 `
 
-const Asset = () => {
+const AssetPage = () => {
   const { tokenId = '', contractAddress = '' } = useParams()
-  const data = useDetailsQuery(contractAddress, tokenId)
+  const { data, loading } = useNftAssetDetails(contractAddress, tokenId)
 
   const [asset, collection] = useMemo(() => data ?? [], [data])
```

Comment:
i'm not sure this memo actually does anything, + it returns an empty array which will be a diff reference anyway